server/views/pages/exhibition.njk

```diff
@@ -124,6 +124,7 @@
               {% if exhibitionContent.textAndCaptionsDocument %}
                 <a href="/download?uri={{ exhibitionContent.textAndCaptionsDocument.url }}"
                   download="{{ exhibitionContent.textAndCaptionsDocument.name }}"
+                  data-track-event='{"category": "component", "action": "download-pdf:click", "label": "name:{{ exhibitionContent.textAndCaptionsDocument.name }}, url:{{ exhibitionContent.textAndCaptionsDocument.url }}"}'
```

Comment:
Could we potentially write a `track-component` function that fills this in?
I am still uncertain what the action is?